validator/client/src/test/java/tech/pegasys/teku/validator/client/VoluntaryExitDataProviderTest.java

```diff
@@ -159,6 +161,16 @@ void createSignedVoluntaryExit_throwsExceptionWhenValidatorNotInList() {
         .hasMessageMatching("Validator (.*) is not in the list of keys managed by this service.");
   }
 
+  @Test
+  void createExitForValidator_notFound() {
+    when(keyManager.getActiveValidatorKeys()).thenReturn(List.of());
```

Comment:
I guess this should rather be `when(keyManager.getActiveValidatorByPublicKey(any())).thenReturn(Optional.empty());`

Reply:
true! I had changed that call late, looks like it still works but i'll clean up the test - thanks.